pkgs/dartpad_ui/lib/widgets.dart

```diff
@@ -79,8 +80,8 @@ class MiniIconButton extends StatelessWidget {
       waitDuration: tooltipDelay,
       child: IconButton(
         style: ButtonStyle(
-          shape: WidgetStateProperty.all(const CircleBorder()),
-          backgroundColor: WidgetStateProperty.all(backgroundColor),
+          shape: const WidgetStatePropertyAll(CircleBorder()),
```

Comment:
I guess `WidgetStateProperty.all` is supposed to be deprecated in favor of this constructor, but a fix is not (or wasn't) possible at the time.